packages/frontend/src/components/DashboardTiles/DashboardChartTile.tsx

```diff
@@ -114,8 +114,15 @@ const DashboardChartTile: FC<Props> = (props) => {
     }
     // END DASHBOARD FILTER LOGIC
 
+    const filterObj =
+        savedQueryWithDashboardFilters?.metricQuery.filters.dimensions;
```

Comment:
This logic is not right because at this point it has the saved chart filters **plus** the dashboard filters. 

So if a saved chart already has its own filter, it will display the message saying it has dashboards filters.